### DISCUSSION TITLE
Settings for Org Browser - which folder to use to retrieve metadata

### DISCUSSION THREAD

**Discussion by tmvillain13** [forcedotcom/salesforcedx-vscode#5761] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm working with several different Salesforce Orgs. These are not different sandboxes, but completely different Production Orgs. 

In my Visual Studio I have 3 different folders with SFDX projects. 

When I try to retrieve meta data with a help of the Org Browser, it is always uses first SFDX project to put meta data into. 

**Describe the solution you'd like**
I would like to have a setting and / or control (on the right click on the Org Browser) with a list of projects. And ability to pick a project, where meta data will be downloaded.

**Comment by lcampos** [comment]
Hi @tmvillain13 thanks for opening this issue. This is one of the things we'll be working on and you should see it added to our roadmap soon. 

For now, there is a way of accomplishing this but it's not ideal since it requires constant configuration changes. The Org Browser feature respects the `packageDirectories` configuration in your project's `sfdx-project.json` file. This means that the metadata will always be retrieved to the `path` you've set as being the default one for your project. One caveat is that updating the default directory takes a few seconds because of some caching involved. We'll be taking care of all of this once we add the functionality to the Org Browser itself.

cc @ntotten
